### DISCUSSION TITLE
Does the Sonoff Zigbee Dongle E as router only need power to work?

### DISCUSSION THREAD

**Discussion by TheZoker** [Koenkk/zigbee2mqtt#25326] [Devices]
Hi there,

I currently have a Sonoff Zigbee Dongle E as my main coordinator directly connected to my home assistant OS running via a proxmox VM. Now I want to switch to a SLZB-06M, which I want to use as my main coordinator. 

To extend my network I want to use the dongle E as a repeater (router). My question now is, do I just need to power the adapter with a power adapter or does it need to be connected to a device to function as a router in my zigbee network? Thanks!

**Comment by chris-1243** [comment]
You need to flash a router firmware first.

Then, plug it to an usb port or charger and pair it to your network

**Reply by TheZoker** [reply]
Ah yes I did forget to mention the flashing the firmware. True
Thanks for the information!
